source/connect/disconnect.txt

```diff
@@ -11,3 +11,16 @@ Disconnect Connection
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+Disconnecting from a MongoDB instance closes the Compass connection to
```

Comment:
[minor]

Suggest replacing "instance" with "deployment" because I think that will bring us more in line with the rest of the docs. Let's try to use "deployment" instead of "instance" where possible when updating the connection docs as a whole.

Reply:
Done.